tasks/README.md

```diff
@@ -1,9 +1,153 @@
-## Tasks List 
+## tasks List 
 
 This directory contains the tasks that are part of this benchmark. 
 
 
 Name | Summary | Category
 ---- | ----------- | --------
-`task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
```

Comment:
This task needs be added as well

Reply:
I have added the required task. Please check now.